sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -159,3 +159,7 @@ ALTER TABLE `product_registration` ADD `last_ask_version` TINYTEXT;
 #IfMissingColumn product_registration options
 ALTER TABLE `product_registration` ADD `options` TEXT COMMENT 'JSON array of scope options';
 #EndIf
+
+#IfTable track_events
+ALTER TABLE `track_events` DROP INDEX `unique_event_label`, ADD UNIQUE `unique_event_label` (`event_label`, `event_url`(255));
```

Comment:
recommend renaming the index on change and instead checking the index via #IfNotIndex
(this way the index won't keep getting reset when run upgrade script more than once)

Reply:
how #IfNotIndex track_events event_url won't work

Reply:
Would try:
```
#IfNotIndex track_events unique_event_label_url
ALTER TABLE `track_events` DROP INDEX `unique_event_label`, ADD UNIQUE `unique_event_label_url` (`event_label`, `event_url`(255));
#EndIf
```

Reply:
fyi, then would need to rename the key in database.sql to unique_event_label_url